includes/Tracking.php

```diff
@@ -90,7 +108,38 @@ public function get_settings() {
 			'trackingAllowed' => $this->is_active(),
 			'trackingId'      => self::TRACKING_ID,
 			'trackingIdGA4'   => self::TRACKING_ID_GA4,
+			// This doesn't seem to be fully working for web properties.
+			// So we send it as both app_version and a user property.
+			// See https://support.google.com/analytics/answer/9268042.
 			'appVersion'      => WEBSTORIES_VERSION,
+			'userProperties'  => $this->get_user_properties(),
+		];
+	}
+
+	/**
+	 * Returns a list of user properties.
+	 *
+	 * @since 1.4.0
+	 *
+	 * @global string $wp_version WordPress version.
+	 *
+	 * @return array User properties.
+	 */
+	private function get_user_properties() {
+		global $wp_version;
+
+		$role        = ! empty( wp_get_current_user()->roles ) ? wp_get_current_user()->roles[0] : '';
+		$experiments = implode( ',', $this->experiments->get_enabled_experiments() );
+
+		return [
+			'siteLocale'         => get_locale(),
+			'userLocale'         => get_user_locale(),
```

Comment:
Does it make sense to use the Location class here?

Reply:
We can keep it simple for only these two values.